libs/shared/package.json

```diff
@@ -39,13 +39,17 @@
   },
   "dependencies": {
     "class-transformer": "0.5.1",
-    "class-validator": "0.14.0"
+    "class-validator": "0.14.0",
+    "ajv": "^8.12.0"
```

Comment:
ajv is safe for both node and browser, however, i wonder if it should be part of the `shared` lib